php/driftphp/config.yaml

```diff
@@ -10,4 +10,4 @@ php_mod:
   - posix
   - sockets
 
-command: php vendor/bin/server run 0.0.0.0:3000 --env=prod --quiet --workers=-1
+command: php vendor/bin/server run 0.0.0.0:3000 --env=prod --quiet --workers=8
```

Comment:
better use a dynamic calculation, no ?

I mean either `nproc` (for the cli) or any trick to calculate the core count in php

Reply:
-1 means "core count" with nproc.

Reply:
do not understand. you set `-1` to use all cores, should be 8 (benchmarks are done a 8 CPU xeon)

Reply:
Exactly, but seems that something is not working. This framework is a high performance PHP framework and is listed almost the last PHP one. These numbers make sense with 1 single core, but not with 8. I will try with specifically 8 cores, so we can detect some kind of issue in that `-1`.

Reply:
I convert this to draft -> meaning (for me), that this requires investigation. I'll try to post you results here, with both configuration

Reply:
This server configuration is set as `--quiet`, which means that there will be nothing in the console. I could add a new option that allow us to know the server header and all errors. That could work,

Reply:
I mean I will output results : 8 workers vs current configuration

Reply:
I updated the PR with this change. Now is not quiet anymore, but we'll be able to see errors and the server header